doc/src/manual/performance-tips.md

```diff
@@ -477,6 +477,12 @@ Here, we happened to know that the first element of `a` would be an [`Int32`](@r
 an annotation like this has the added benefit that it will raise a run-time error if the
 value is not of the expected type, potentially catching certain bugs earlier.
 
+A slightly more generic way to annotate the type of `x` is `x = convert(Int32,a[1])::Int32`.  The
```

Comment:
The second part comes through pretty well but your response to me here, if worked into what you already have, would make it a lot clearer I think. Thanks for adding this!